homeassistant/components/miele/strings.json

```diff
@@ -114,7 +114,659 @@
+          "cauliflower_whole_medium": "Cauliflower (whole, medium)",
+          "cauliflower_whole_small": "Cauliflower (whole, small)",
+          "celeriac_cut_into_batons": "Celeriac (cut into batons)",
+          "celeriac_diced": "Celeriac (diced)",
+          "celeriac_sliced": "Celeriac (sliced)",
+          "celery_pieces": "Celery (pieces)",
+          "celery_sliced": "Celery (sliced)",
+          "cep": "Cep",
+          "chanterelle": "Chanterelle",
+          "char": "Char",
+          "check_appliance": "Check appliance",
+          "cheesecake_one_large": "Cheesecake (one large)",
+          "cheesecake_several_small": "Cheesecake (several small)",
+          "chick_peas": "Chick peas",
+          "chicken_tikka_masala_with_rice": "Chicken Tikka Masala with rice",
+          "chinese_cabbage_cut": "Chinese cabbage (cut)",
+          "chongming_rapid_steam_cooking": "Chongming (rapid steam cooking)",
+          "chongming_steam_cooking": "Chongming (steam cooking)",
+          "christmas_pudding_cooking": "Christmas pudding (cooking)",
+          "christmas_pudding_heating": "Christmas pudding (heating)",
+          "clean_machine": "Clean machine",
+          "coalfish_fillet_2_cm": "Coalfish (fillet, 2 cm)",
+          "coalfish_fillet_3_cm": "Coalfish (fillet, 3 cm)",
+          "coalfish_piece": "Coalfish (piece)",
+          "cockles": "Cockles",
+          "codfish_fillet": "Codfish (fillet)",
+          "codfish_piece": "Codfish (piece)",
+          "coffee": "Coffee",
+          "coffee_pot": "Coffee pot",
+          "common_beans": "Common beans",
+          "common_sole_fillet_1_cm": "Common sole (fillet, 1 cm)",
+          "common_sole_fillet_2_cm": "Common sole (fillet, 2 cm)",
+          "conventional_heat": "Conventional heat",
+          "cook_bacon": "Cook bacon",
+          "cool_air": "Cool air",
+          "corn_on_the_cob": "Corn on the cob",
+          "cottons": "Cottons",
+          "cottons_eco": "Cottons ECO",
+          "cottons_hygiene": "Cottons hygiene",
+          "courgette_diced": "Courgette (diced)",
+          "courgette_sliced": "Courgette (sliced)",
+          "cranberries": "Cranberries",
+          "crevettes": "Crevettes",
+          "curtains": "Curtains",
+          "dark_garments": "Dark garments",
+          "decrystallise_honey": "Decrystallise honey",
+          "defrost": "Defrost",
+          "defrosting_with_microwave": "Defrosting with microwave",
+          "defrosting_with_steam": "Defrosting with steam",
+          "delicates": "Delicates",
+          "denim": "Denim",
+          "descale": "Descale",
+          "descaling": "Appliance descaling",
+          "dissolve_gelatine": "Dissolve gelatine",
+          "down_duvets": "Down duvets",
+          "down_filled_items": "Down-filled items",
+          "drain_spin": "Drain/spin",
+          "dutch_hash": "Dutch hash",
+          "eco": "ECO",
+          "eco_40_60": "ECO 40-60",
```

Comment:
Any reason why ECO is capitalized at a few and not on the others

Reply:
The strings have been harvested by a user and I dont know how he got to this mix. I know for sure that "ECO" and "ECO 40-60" are used by Miele. I change the others.